### DISCUSSION TITLE
Unable to change the colour in Most Used Languages card

### DISCUSSION THREAD

**Discussion by APratham** [anuraghazra/github-readme-stats#953] [Q&A] [answered]
I have been able to customise the colour of the Most Used Languages card, but I wanted to know if it is possible to change the colour completely, maybe make it one solid colour. If you wanted to see what I mean, feel free to visit my GitHub profile README

**Comment by anuraghazra** [comment, marked as answer]
Of course it's possible. 

Read the docs please https://github.com/anuraghazra/github-readme-stats#common-options

Live demo 

https://github-readme-stats.vercel.app/api/top-langs/?username=anuraghazra&text_color=fff&bg_color=258&title_color=fff

**Comment by Nik6511** [comment]
Yes, you definitely can

**Comment by guddukrp** [comment]
Thanks

**Comment by arthurr-jpg** [comment]
Yes, you definitely can

**Comment by genze121** [comment]
> I have been able to customise the colour of the Most Used Languages card, but I wanted to know if it is possible to change the colour completely, maybe make it one solid colour. If you wanted to see what I mean, feel free to visit my GitHub profile README

Yes, it's easily customizable.

**Comment by arthurr-jpg** [comment]
Nice job
